apps/studio/src/lib/db/clients/postgresql.ts

```diff
@@ -1502,6 +1503,10 @@ async function runWithConnection<T>(x: Conn, run: (p: PoolClient) => Promise<T>)
   }
 }
 
+function getVersionString(version: VersionInfo): string {
+  return version.version.split(" on ")[0];
```

Comment:
Does this work properly for Cockroach DB and Redshift do you think? Maybe it should return the full version for those?